react/components/SKUSelector/components/SelectorItem.js

```diff
@@ -89,17 +95,4 @@ SelectorItem.propTypes = {
   isImage: PropTypes.bool,
 }
 
-SelectorItem.defaultProps = {
-  index: 0,
-  children: {},
-  isAvailable: true,
-  isSelected: false,
-}
-
-const getDiscount = (maxPrice, price) => {
-  let discount = 0
-  if (maxPrice && price) {
-    discount = 1 - price / maxPrice
-  }
-  return discount
-}
+export default memo(SelectorItem)
```

Comment:
is it smart to use memo on a component that makes use of the "children" prop? see thread: https://github.com/facebook/react/issues/14463

Reply:
I just refactored it to a function component, and since it was a pure component before I added this hoc, but good call :+1: